packages/core-cordova/www/metrics.js

```diff
@@ -0,0 +1,57 @@
+const MetricsService = cordova.require("core.core").MetricsService;
+
+const MOBILE_CORE_CLASS = "MobileCoreModule";
+
+class MetricsImpl {
+
+  constructor(id, data) {
+    this.id = id;
+    this.data = data;
+  }
+
+  collect() {
+    return this.data;
+  }
+}
+
+class CordovaMetricsService extends MetricsService {
+
+  getAppAndDeviceMetrics() {
+    return new Promise((res, rej) => {
+      cordova.exec(
+        metrics => {
+          const plugins = cordova.require("cordova/plugin_list");
+          const sdkVersion = plugins.metadata["core-cordova"];
+          metrics.app.sdkVersion = sdkVersion;
+          res(metrics);
+        },
+        rej,
+        MOBILE_CORE_CLASS,
+        "getMetrics"
+      );
+    });
+  }
+
+  sendAppAndDeviceMetrics() {
+    return this.getAppAndDeviceMetrics()
+      .then(metrics => {
+        const appMetrics = new MetricsImpl("app", metrics.app);
+        const deviceMetrics = new MetricsImpl("device", metrics.device);
+
+        return this.publish([appMetrics, deviceMetrics]);
+      });
+  }
+
+  getSavedClientId() {
+    // TODO: get from device storage or return undefined
+    return undefined;
```

Comment:
Do we have ticket to do it? if not let's implement that here.

Reply:
Don't you think we can start branching some of these todo items to tickets? I think getting the main structure merged should be a priority.

Reply:
If we look at the roadmap, getting the app metrics working is in a [future card](https://trello.com/c/p2gBN9XB/48-5-cordova-appmetrics-as-a-mobile-developer-i-want-the-cordova-sdk-to-give-me-app-launch-metrics-such-that-i-can-visualise-and-an), not saying we can't speed things up, of course :)

Reply:
@paolobueno Good point! Let's create tickets to make sure that will be covered.